<launch>
	<node name="xbot_talker" pkg="xbot_talker" type="talker">
		<param name="base_path" value="$(find xbot_talker)" />
	</node>
</launch>
